alacritty/src/event.rs

```diff
@@ -214,30 +219,71 @@ impl Processor {
                 | WindowEvent::DoubleTapGesture { .. }
                 | WindowEvent::TouchpadPressure { .. }
                 | WindowEvent::RotationGesture { .. }
-                | WindowEvent::CursorEntered { .. }
+                | WindowEvent::PointerEntered { .. }
                 | WindowEvent::PinchGesture { .. }
-                | WindowEvent::AxisMotion { .. }
+                | WindowEvent::DragEntered { .. }
                 | WindowEvent::PanGesture { .. }
-                | WindowEvent::HoveredFileCancelled
+                | WindowEvent::DragMoved { .. }
+                | WindowEvent::DragLeft { .. }
                 | WindowEvent::Destroyed
                 | WindowEvent::ThemeChanged(_)
-                | WindowEvent::HoveredFile(_)
                 | WindowEvent::Moved(_)
         )
     }
 }
 
-impl ApplicationHandler<Event> for Processor {
-    fn resumed(&mut self, _event_loop: &ActiveEventLoop) {}
+impl Drop for Processor {
+    fn drop(&mut self) {
+        if self.config.debug.print_events {
```

Comment:
This should only really affect our winit events, why is it used for one of our own events? Seems inconsistent/confusing.